src/librustc_driver/pretty.rs

```diff
@@ -783,32 +813,36 @@ pub fn pretty_print_input(sess: Session,
                                                         &id,
                                                         resolve::MakeGlobMap::No,
                                                         |tcx, _, _| {
-                        print_flowgraph(variants, tcx, code, mode, out)
-                    })
+                                                            print_flowgraph(variants,
+                                                                            tcx,
+                                                                            code,
+                                                                            mode,
+                                                                            out)
+                                                        })
                 }
                 None => {
-                    let message = format!("--pretty=flowgraph needs \
-                                           block, fn, or method; got {:?}",
+                    let message = format!("--pretty=flowgraph needs block, fn, or method; got \
+                                           {:?}",
                                           node);
 
                     // point to what was found, if there's an
                     // accessible span.
                     match ast_map.opt_span(nodeid) {
                         Some(sp) => sess.span_fatal(sp, &message[..]),
-                        None => sess.fatal(&message[..])
+                        None => sess.fatal(&message[..]),
                     }
                 }
             }
         }
-    }.unwrap();
+    }
+    .unwrap();
```

Comment:
it's kind of a corner case - with a smaller block or a chain of method calls I think it looks good, but this is a bit weird